components/workspace/Menu.vue

```diff
@@ -33,8 +33,12 @@ defineEmits<{
   (e: 'workspace:selected', result: WorkspaceListWorkspaceItemFragment): void
 }>()
 
+const completedWorkspaces = computed(() =>
+  props.workspaces.filter((w) => w.creationState?.completed !== false)
+)
+
 const workspacesWithPersonalProjects = computed(() => [
```

Comment:
```
const workspacesWithPersonalProjects = computed(() => [
  ...props.workspaces.filter((w) => w.creationState?.completed !== false),
  {
    id: 'personalProject',
    name: 'Personal Projects'
```

and you do not need the computed above since it's not used anywhere else?